src/main/resources/io/jenkins/plugins/restlistparam/RestListParameterDefinition/index.jelly

```diff
@@ -20,7 +20,7 @@
       <select id="${id}-select" class="setting-input" name="value">
         <j:forEach var="value" items="${it.values}" varStatus="loop">
           <f:option value="${value}" selected="${value.equals(it.defaultValue)}">
```

Comment:
Proposal, since I am not such a big fan of the stream logic introduced in the pram definition to create a RLPV, use the index of the value instead of the value itself and then simply access the list at the index to get which value was selected?

Reply:
I like it, I'll give it a shot, possibly tonight, or sunday

Reply:
I really need this consultation, because I'm no expert in writing jenkins plugins.

Reply:
we all have to start somewhere, if you are stuck somewhere feel free to ping me! I may not be an expert myself, for that Jenkins is too big and its dev documentation a bit too ... yes, however, I may still be able to help ;)